Code/Framework/AzCore/AzCore/RTTI/RTTI.h

```diff
@@ -977,26 +977,32 @@ namespace AZ
     {
         return AzGenericTypeInfo::Uuid<U>();
     }
-    
+
+    #if defined(AZ_COMPILER_MSVC)
+    // There is a bug with the MSVC compiler when using the 'auto' keyword here. It appears that MSVC is unable to distinguish between a template
+    // template argument with a type variadic pack vs a template plate argument with a non-type auto variadic pack. 
```

Comment:
typo: template plate -> template template

```suggestion
    // template argument with a type variadic pack vs a template template argument with a non-type auto variadic pack. 
```